test/functional/pdo_sqlsrv/pdo_937_metadata.phpt

```diff
@@ -0,0 +1,115 @@
+--TEST--
+GitHub issue 937 - getting metadata will not fail after an UPDATE / DELETE statement
+--DESCRIPTION--
+Verifies that getColumnMeta will not fail after processing an UPDATE / DELETE query that returns no fields.
+--ENV--
+PHPT_EXEC=true
+--SKIPIF--
+<?php require('skipif_mid-refactor.inc'); ?>
+--FILE--
+<?php
+require_once("MsSetup.inc");
+require_once("MsCommon_mid-refactor.inc");
+
+$tableName = 'pdoTestTable_938';
+$procName = 'pdoTestProc_938';
+
+try {
+    $conn = connect();
+    
+    dropTable($conn, $tableName);
+    dropProc($conn, $procName);
+
+    $tsql = "CREATE TABLE $tableName([id] [int] NOT NULL, [name] [varchar](10) NOT NULL)";
+    $conn->query($tsql);
+    
+    $id = 3;
+    $tsql = "INSERT INTO $tableName VALUES ($id, 'abcde')";
+    $conn->query($tsql);
+    
+    $tsql = "UPDATE $tableName SET name = 'updated' WHERE id = $id";
+    $stmt = $conn->prepare($tsql);
+    $stmt->execute();
+    $numCol = $metadata = $stmt->columnCount();
+    echo "Number of columns after UPDATE: $numCol\n";
+
+    $tsql = "SELECT * FROM $tableName";
+    $stmt = $conn->query($tsql);
+    $numCol = $metadata = $stmt->columnCount();
+    for ($i = 0; $i < $numCol; $i++) {
+        $metadata = $stmt->getColumnMeta($i);

```

Comment:
This is the only place that getColumnMeta is called? In the sqlsrv test sqlsrv_field_metadata is called several times for both non-empty and empty result sets. Is it preferable to call columnCount first in PDO?